### PR TITLE
Add Element that renders nothing

### PR DESCRIPTION
**Pull request by sshihata**
Added an element that can be used to render nothing on the failure of some
condition. #90

### CODE REVIEW

**Comment by fbonetti** [comment]
This is pedantic, but I would reword the comment.

**Current:**
```
Represents rendering nothing when some condition is not true.
```

**Revised:**
```
Represents rendering nothing. Useful for when you want to optionally render an element.
```

The current definition makes it sound as if `nothing` takes a `Bool` as an argument

**Comment by sshihata** [comment]
Bad wording on my part. Fixed !
